.github/ISSUE_TEMPLATE/design_story.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "Design story \U0001F5E3"
+about: Write a user story to begin solving their needs.
+title: ''
+labels: ''
+assignees: RobertaJHahn, ljcarot, wonilsuhibm
+
+---
+
+<!-- Avoid any type of solutions in this user story -->
+<!-- replace _{{...}}_ with your own words -->
+
+# As a `[user]`, I need `[goal]` so that `[measurable insight]`
+ _{{Provide a detailed description of the user's need here, but avoid any type of solutions}}_
+
+## Additional information
+ _{{Please provide any additional information or resources for reference}}_
+ - _{{user research}}_
+ - _{{user insights}}_
+ - _{{user metrics}}_
+
+## Acceptance criteria
+- [ ] Design completed
+- [ ] Design posted on Wake for general feedback
+- [ ] Design shared with IDL for IDL feedback
+- [ ] Design accepted by stakeholders
+- [ ] Design reviewed and approved by dev team
+- [ ] Design approved by DGC design leadership (Charlie Douglas, Wonil Suh, Elliott Trice)
```

Comment:
@oliviaflory @wonilsuhibm wondering if we want to keep names as part of this template since this is open source?